_integration/testsuite/httpproxy/004-https-sni-enforcement.yaml

```diff
@@ -0,0 +1,370 @@
+# Copyright 2020 VMware, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.  You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import data.contour.resources
+
+# Ensure that cert-manager is installed.
+# Version check the certificates resource.
+
+Group := "cert-manager.io"
```

Comment:
Should this stuff be in a generic top level set of code? I assume we'll need cert-manager in this example in other tests. How do we keep all this in sync?

Reply:
There's a trade-off between minimizing copy/paste and keeping things obvious, and I mostly haven't minded a bit of extra copy/paste so far. Maybe one way to approach this particular issue is for tests to specify a table of GVRs that they will need, then a library function to do the set difference for the skip query?

Reply:
We could probably reduce this to
```
skip[msg] {
   not resources.is_supported_version("certificates", "cert-manager.io/v1alpha2")
   msg := "cert-manager is not installed"
 }
```